server/channels/db/migrations/postgres/000138_add_remoteid_channelid_to_post_acknowledgements.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE postacknowledgements ADD COLUMN remoteid VARCHAR(26) DEFAULT '';
+ALTER TABLE postacknowledgements ADD COLUMN channelid VARCHAR(26) NOT NULL DEFAULT '';
+
+UPDATE postacknowledgements pa
+SET channelid = p.channelid
+FROM posts p
+WHERE pa.postid = p.id
+AND pa.channelid = '';
+
+ALTER TABLE postacknowledgements ALTER COLUMN channelid DROP DEFAULT;
```

Comment:
Unfortunately, we cannot do this without breaking our non-blocking schema migrations rule: https://developers.mattermost.com/contribute/more-info/server/schema-migration-guide.

If you really want to do this, follow this approach:

https://developers.mattermost.com/contribute/more-info/server/schema-migration-guide/#how-do-i-run-update-statements-in-data-migrationsdenormalizations

Regarding the `DROP DEFAULT` command, can you explain the idea behind that? It seems to me like we are adding a default, then setting the values, and then dropping the default. What is the difference from not setting `DEFAULT ''` in the first place?

Reply:
@catalintomai how can we better signal when a PR is up for reviews vs still in works? With the PR reviewers assigned, I was also under the impression that the PR was ready for reviews.

Reply:
Thanks @catalintomai. If the PR is still in works, I'd expect it to be in a draft state. The CODEOWNERS file respects that and won't add reviewers until a PR is ready.

Reply:
Thanks @agnivade . I was using the "in work" label to move in/out of state, but I'll use the draft mode going forward.